app/helpers/alchemy/elements_block_helper.rb

```diff
@@ -26,12 +26,13 @@ def element
     class ElementViewHelper < BlockHelper
       # Renders one of the element's contents.
       #
+      # If the element uses +ingredients+ it renders the ingredient record.
+      #
       def render(name, options = {}, html_options = {})
-        content = element.content_by_name(name)
-        return if content.nil?
+        renderable = content(name) || element.ingredient_by_role(name)
```

Comment:
This should probably be reversed. After running the rake task, I tried changing a picture element, and the change wouldn't show up on the page unless the ingredient took precedence.

Reply:
Absolutely, yes!